site/docs/concepts/architecture/control-plane.md

```diff
@@ -14,7 +14,7 @@ The control plane is responsible for configuring and managing the system. It con
 
 The control plane operates through a chain of components that work together to manage the configuration:
 
-1. The Envoy AI Gateway controller watches the AI Gateway CRD
+1. The Envoy AI Gateway controller watches `AIGatewayRoute` custom resources
```

Comment:
Is it just this one resource or all `aigateway.envoyproxy.io` K8s custom resources?

Reply:
nope, there are three resources https://aigateway.envoyproxy.io/docs/api/#available-kinds

Reply:
ah i see that further down the page too, thanks

```suggestion
1. The Envoy AI Gateway controller watches AI Gateway Custom Resources (CRs)
```